Build.Docker.ps1

```diff
@@ -50,6 +50,20 @@ function Publish-DockerImage($arch)
     $ErrorActionPreference = "Stop"
 }
 
+function Publish-DockerManifest($archs)
+{
+    $images = ""
+    foreach ($arch in $archs) {
+        $images += "$image-ci:$version-$($arch.rid) "
+    }
+
+    invoke-expression "docker manifest create $image-ci:$version $images"
```

Comment:
Guessing this is because Docker really wants the `$images` items as separate arguments; might be worth a quick comment?